<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screenEsploraDocumenti</class>
 <widget class="QWidget" name="screenEsploraDocumenti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1618</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="button_cambiaLogin">
         <property name="text">
          <string>Cambia utente</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_utenteLog">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="w_toolBar" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="layoyt_toolbar"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="1" column="0">
      <widget class="QTableView" name="tableViewEsploraDoc">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_9" rowstretch="0">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelTotale">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>Utile:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="labelTotale_2">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>Spese:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="l_viewTotaleEntrate">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelTotale_3">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>Entrate:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="l_viewTotaleSpese">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="l_viewTotaleDoc">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="tb_salvaCsv">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filtri</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QToolButton" name="tb_cercaDett">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="r_testata">
          <property name="text">
           <string>Testata</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="r_righe">
          <property name="text">
           <string>Righe</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string>Id testata:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_cercaIdTestata">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QDateEdit" name="dataEditCercaDa"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string>Da:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dataEditCercaDa_2"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string>A:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string>Tipo documento:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_cercaTipoDoc">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="tb_tipoDoc">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="layoutDettaglioDoc"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_cambiaLogin</tabstop>
  <tabstop>tableViewEsploraDoc</tabstop>
  <tabstop>tb_cercaDett</tabstop>
  <tabstop>l_cercaIdTestata</tabstop>
  <tabstop>dataEditCercaDa</tabstop>
  <tabstop>dataEditCercaDa_2</tabstop>
  <tabstop>l_viewTotaleDoc</tabstop>
  <tabstop>l_viewTotaleEntrate</tabstop>
  <tabstop>l_viewTotaleSpese</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
